src/screens/VaultScreen.js

```diff
@@ -109,14 +150,40 @@ export default class VaultScreen extends React.Component {
   /**
    * Upon call, returns the filter category elements based on the category array in the state.
    */
```

Comment:
can you change this comment to reflect the function below it?

Reply:
@clairelin135 is this comment related to the upvotes method?

Reply:
no, this is just creates and returns the filter buttons based on the state